standards/rmrk2.0.0/interactions/changeissuer.md

```diff
@@ -0,0 +1,63 @@
+# CHANGEISSUER
+
+The CHANGEISSUER interaction allows a [Base](../entities/base.md) or
+[Collection](../entities/collection.md) issuer to change the issuer value to another address. The
+original issuer immediately loses all rights to further interact with the entity.
+
+This is particularly useful when another team is taking control over a project, or if an address was
+compromised.
+
+## Standard
+
+The format of a CHANGEISSUER interaction is
+`0x{bytes(rmrk::CHANGEISSUER::{version}::{id}::{newissuer})}`.
+
+- `version` is the version of the standard used (e.g. `2.0.0`)
+- `id` is the [base](../entities/base.md)'s ID or the [collection](../entities/collection.md)'s ID..
+- `newissuer` is the address of the new owner, e.g.
+  `CpjsLDC1JFyrhm3ftC9Gs4QoyrkHKhZKtK7YqGTRFtTafgp`
+
+## Examples
+
+### Base
+
+If we want to change the owner of Base `kanaria-epic-birds` to

```

Comment:
Maybe good to specify computed id here `base-{block}-{id}` so the person reading this doc knew how to differentiate between CHANGEISSUER for base and CHANGEISSUER for nftclass